keyboards/ergodash/rev2/keymaps/vishnu/keymap.c

```diff
@@ -0,0 +1,116 @@
+    _______, _______, _______, _______, _______, _______,_______,                       _______, KC_LEFT, KC_DOWN, KC_UP  , KC_RIGHT, _______, _______, \
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______,  _______, _______ , \
+    _______, _______, _______,           KC_LBRC,_______,KC_ENT ,_______,       _______,KC_ENT , _______, KC_RBRC,          _______,  _______, OSL(_FKeys)     \
+  ),
+  /* FKeys
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   * |      |  F1  |  F2  |  F3  |  F4  |  F5  | F11  |                    | F12  |  F6  |  F7  |  F8  |  F9  | F10  |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |------+------+------+------+------+------+---------------------------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |                    |      |      |      |      |      |      |      |
+   * |-------------+------+------+------+------+------+------+------+------+------+------+------+------+------+-------------|
+   * |      |      |      ||||||||      |      |      |      ||||||||      |      |      |      ||||||||      |      |      |
+   * ,----------------------------------------------------------------------------------------------------------------------.
+   */
+  [_FKeys] = LAYOUT(
+    _______,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5, KC_F11,                        KC_F12,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  _______ , \
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______,  _______, _______ , \
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______,  _______, _______ , \
+    _______, _______, _______, _______, _______, _______,_______,                       _______, _______, _______, _______, _______,  _______, _______ , \
+    _______, _______, _______,          _______, _______,_______,_______,       _______,_______, _______, _______,          _______,  _______, _______   \
+  )
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer